appveyor.yml

```diff
@@ -25,7 +25,7 @@ install:
   - conda install statsmodels
   - pip install Pillow
   - conda install -c https://conda.anaconda.org/simpleitk SimpleITK
-  - if %PYTHON% == 2.7 conda install scikit-image
+  - if %PYTHON% == 2.7 conda install -c conda-forge scikit-image
```

Comment:
Mabye a quick PR on allensdk/master to update there until this branch is merged

Reply:
good point. Here: https://github.com/AllenInstitute/AllenSDK/pull/408